### ISSUE TITLE
pbm hangs when connecting to lighthouse deployed in AWS ECS

### ISSUE THREAD

**Issue by rafalpiotrowski** [petabridge/petabridge.cmd-issues#48]
I am running Lighthouse service in AWS ECS Fargate container. When I call pbm myfargateservice:9110 the pbm is able to establish connection but it does not go to the prompt but instead hangs after 
PS C:\> pbm xyz.amazonaws.com:9110
 ____      _        _          _     _
|  _ \ ___| |_ __ _| |__  _ __(_) __| | __ _  ___
| |_) / _ | __/ _` | '_ \| '__| |/ _` |/ _` |/ _ \
|  __|  __| || (_| | |_) | |  | | (_| | (_| |  __/
|_|   \___|\__\__,_|_.__/|_|  |_|\__,_|\__, |\___|
                                       |___/

petabridge.cmd (1.1.0.0)
Copyright 2017 - 2022, Petabridge®.

NOTHING IS SHOWING, I don't see errors in logs either...

When running locally in docker, the pbm is able to connect to the service and is fine.

Is there some sort of extra settings that need to be set/changes on the netwrok level?

**Comment by Aaronontheweb** [comment]
Yeah I think this is related to https://github.com/petabridge/petabridge.cmd-issues/issues/45 - we probably need to disable our ASCII art when running on Linux. Not entirely sure what the issue is there.
